locales/ja-JP.yml

```
@@ -425,7 +425,7 @@ quoteQuestion: "引用として添付しますか？"
 noMessagesYet: "まだチャットはありません"
 newMessageExists: "新しいメッセージがあります"
 onlyOneFileCanBeAttached: "メッセージに添付できるファイルはひとつです"
-signinRequired: "ログインしてください"
+signinRequired: "続行する前に、サインアップまたはサインインが必要です"
```

Using the proper japanese word for registering and login instead of sign-in here seems better to me.
Rephrased to "Please register or login to continue", as that is closer to the previous as well.

```suggestion
signinRequired: "進むために、登録またはログインしてください"
```

Would nonetheless wait for input from a maintainer on that phrasing.